packages/server-wallet/src/protocols/process-ledger-queue.ts

```diff
@@ -134,16 +134,13 @@ const xorOutcome = (a: SimpleAllocation, b: SimpleAllocation): SimpleAllocation
  *
  * IF there are any pending ledger updates (to fund OR defund a channel)
  *
- *   i. Check if I already signed a new ledger update O₁. If I did, then use that. [1]
+ *   i. Check if I already signed a new ledger update. If I did, then call this O₁. [1]
  *      Otherwise, iteratively allocate funds from the supported state's (i.e., S) outcome
  *      to each request, call this O₁.
  *
- *  ii. Check if there is a conflicting ledger update, O₂.
- *
- *      a. If O₁ ⋂ O₂ ≠ O₁ sign an outcome for O₁ ⋂ O₂ with turn number 2
- *         higher than the one corresponding to O₂.
- *
- *      b. If O₁ ⋂ O₂ = O₁ = O then sign O with turn number 2 corresponding to O₂.
+ *  ii. If there is a conflicting ledger update, call that O₂. Sign O₁ ⋂ O₂ with the higher turn
```

Comment:
```suggestion
 *  ii. If there is a conflicting ledger update, call its outcome O₂. Sign a state with outcome O₁ ⋂ O₂ and the higher turn number
```

I really like the mathematical notation in the code, but we **also need to define** what O₁ ⋂ O means.

Also, what is "the higher turn number"?